src/index.js

```diff
@@ -161,9 +164,9 @@ function jumpToPast (history, index) {
 
 // jump: jump n steps in the past or forward
 function jump (history, n) {
-  if (n === 0) return history
   if (n > 0) return jumpToFuture(history, n - 1)
   if (n < 0) return jumpToPast(history, history.past.length + n)
+  return history
```

Comment:
coverage said the else of the last if was never called.. which was correct.. but also a bit bullshit. Anyway, this is better since users will probably use `jump(0)` the least amount of times